docs/_docs/04_community/client-factory.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: collection-browser-doc
+title: Azure SDK Client Factory
+category: contributing
```

Comment:
This ends up as a totally new top-level nav category, with solely this Azure stuff in it, which probably isn't what we want:

<img width="1036" alt="Screen Shot 2020-11-17 at 5 11 15 PM" src="https://user-images.githubusercontent.com/711908/99422957-f78a4800-28f7-11eb-8729-5923935c09ff.png">

Note that we have an existing contributing page already under the Community nav:

<img width="395" alt="Screen Shot 2020-11-17 at 5 12 52 PM" src="https://user-images.githubusercontent.com/711908/99423127-27d1e680-28f8-11eb-81fd-6a019b6ce6cf.png">

Probably the easiest thing to do would be to move the content of this page into that existing Contributing page, just below the "Developing Terratest" section.

Reply:
Moved content to "Contributing" section as called out.